rpm/embedlite-components-qt5.spec

```diff
@@ -1,3 +1,20 @@
+# Don't depend on private xulrunner-qt5 libraries.
+%global privlibs             libfreebl3
+%global privlibs %{privlibs}|libmozalloc
+%global privlibs %{privlibs}|libmozsqlite3
+%global privlibs %{privlibs}|libnspr4
+%global privlibs %{privlibs}|libnss3
+%global privlibs %{privlibs}|libnssdbm3
+%global privlibs %{privlibs}|libnssutil3
+%global privlibs %{privlibs}|libplc4
+%global privlibs %{privlibs}|libplds4
+%global privlibs %{privlibs}|libsmime3
+%global privlibs %{privlibs}|libsoftokn3
+%global privlibs %{privlibs}|libssl3
+
+%global __provides_exclude ^(%{privlibs})\\.so
```

Comment:
I think that this line doesn't makes much sense for embedlite-components-qt5 as this package doesn't reproduce private libs as build results, this package only requires them.

> rpm -q --requires embedlite-components-qt5
> rpm -q --provides embedlite-components-qt5

Are same with and without provides_exlude.